src/pages/Loans/LoansOverview/components/LoanForm/LoanForm.tsx

```diff
@@ -128,6 +128,8 @@ const LoanForm = ({ asset, variant, position, onChangeLoan }: LoanFormProps): JS
   const [isMaxAmount, setMaxAmount] = useState(false);
 
   const handleSuccess = () => {
+    const successText = variant === 'lend' ? 'Lending' : 'Withdrawal';
```

Comment:
This is an unrelated change (and has been superseded)—could it be removed please?